stable/firehose/templates/_helpers.tpl

```diff
@@ -39,6 +39,15 @@
   write_consistency = "any"
   timeout = "5s"
 {{- end }}
+{{- if .Values.telegraf.config.output.prometheus_remote_write.enabled }}
+[[outputs.http]]
+  url = {{.Values.telegraf.config.output.prometheus_remote_write.url}}
+  data_format = "prometheusremotewrite"
+  [outputs.http.headers]
+      Content-Type = "application/x-protobuf"
+      Content-Encoding = "snappy"
+      X-Prometheus-Remote-Write-Version = {{.Values.telegraf.config.output.prometheus_remote_write.version}}
```

Comment:
version string seems to be missing.

Reply:
sorry, missing where? 🤔 `version` config is there in the values.yaml

Reply:
Weird. I wasn't seeing anything there! My bad.